### PR TITLE
Clarifying the purpose and intended use of a ConfigMap

### PR DESCRIPTION
**Pull request by thesteve0**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1363)

### CODE REVIEW

**Comment by thesteve0** [comment]
Not sure how to associate this pull request with my Red Hat credentials. spousty@redhat.com

File: docs/user-guide/configmap/index.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ or used to store configuration data for system components such as controllers.
 to [Secrets](/docs/user-guide/secrets/), but designed to more conveniently support working with strings that do not
 contain sensitive information.
 
+Note: ConfigMaps are not intended to act as a replacement for a properties file. ConfigMaps are more intended to act a reference to multipe propertie files. You can think of them as way to represent something similar to the /etc directory and all it's files on a Linux computer. This model for ConfigMaps becomes especially apparent when looking at creating Volumes from ConfigMaps. Each data item in the ConfigMap becomes a new file.
```
Comment (eparis) [inline docs/user-guide/configmap/index.md]: `propertie` typo.

**Comment by thesteve0** [comment]
@eparis Anything I should do to move this along?

**Comment by eparis** [comment]
Fix the typo and it seems you need to rebase.
You should probably sign the CNCF CLA, although that is not strictly necessary to most this along. But it will be soon.

File: docs/user-guide/configmap/index.md (hunk 1 above)
Comment (jaredbhatti) [inline docs/user-guide/configmap/index.md]: Also, "multiple" instead of "multipe".  I'd reword the sentence to: " ConfigMaps are intended to act a reference to multiple property files"

Comment (jaredbhatti) [inline docs/user-guide/configmap/index.md]: also "its" instead of "it's"

**Comment by jaredbhatti** [comment]
Small docs issues.  Please update and I'll merge.

**Comment by devin-donnelly** [comment]
Fixed the wording with an additional commit. I'll merge.
